### PR TITLE
Fix the script for AWS and GCP depending on the ssh user permission

### PR DESCRIPTION
**Pull request by alallema**
Depending on the user with whom you will connect to your instance to launch your script you will not necessarily have access to the api key nor the same rights to delete some files and folders
This PR will notice the user about it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,22 @@ sh migration.sh meilisearch_version
 
 - `meilisearch_version`: the Meilisearch version formatted like this: `vX.X.X`
 
+**Note**
+
+If you want to run the script from an AWS instance and you are logged in as `admin`, you will probably have to use this command instead:
```
Comment (bidoubiwa) [inline README.md:95]: > you will probably

Probably or certainly ?

```suggestion
If you want to run the script from an AWS instance and you are logged in as `admin`, you probably have to use this command instead:
```

No need to use `will`

File: README.md [hunk 2]
```diff
@@ -90,6 +90,22 @@ sh migration.sh meilisearch_version
...
+```bash
+sudo -E sh migration.sh meilisearch_version
+```
+
+If you want to run the script from a GCP VM instance and you are logged in as a user, you will probably have to set the $MEILISEARCH_MASTER_KEY like:
```
Comment (bidoubiwa) [inline README.md:100]:
```suggestion
If you want to run the script from a GCP VM instance and you are logged in as a user, you probably have to set the $MEILISEARCH_MASTER_KEY like:
```

File: scripts/update_meilisearch_version.sh [hunk 3]
```diff
@@ -104,12 +104,21 @@ check_last_exit_status() {
     fi
 }
 
+#Check if the Api key is available is the environment variable
+check_api_key() {
+    if [ -z $MEILISEARCH_MASTER_KEY ]; then
+        echo "${WARNING_LABEL}MEILISEARCH_MASTER_KEY is not available in the environment variables if you have an api key you must set it with the following command\nexport MEILISEARCH_MASTER_KEY=YOUR_API_KEY"
+    fi
+}
```
Comment (bidoubiwa) [inline scripts/update_meilisearch_version.sh:112]: Should we `exit` the script if it fails?

Comment (bidoubiwa) [reply scripts/update_meilisearch_version.sh:112]:
```suggestion
check_api_key() {
    if [ -z $MEILISEARCH_MASTER_KEY ]; then
        echo "${WARNING_LABEL}MEILISEARCH_MASTER_KEY is not available in the environment variables. If you have an API key, set it with the following command 'export MEILISEARCH_MASTER_KEY=YOUR_API_KEY'"
    fi
}
```

File: scripts/update_meilisearch_version.sh [hunk 4]
```diff
@@ -104,12 +104,21 @@ check_last_exit_status() {
...
+
 ## Main Script
 
 #
 # Current Running Meilisearch
 #
 
+check_api_key
```
Comment (bidoubiwa) [inline scripts/update_meilisearch_version.sh:120]:
```suggestion
# Confirm that the API key is defined in the env variables.
check_api_key
```

File: scripts/update_meilisearch_version.sh [hunk 5]
```diff
@@ -104,12 +104,21 @@ check_last_exit_status() {
     fi
 }
 
+#Check if the Api key is available is the environment variable
```
Comment (bidoubiwa) [inline scripts/update_meilisearch_version.sh:107]:
```suggestion
# Check if the API key is defined is the environment variable
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

File: scripts/update_meilisearch_version.sh (hunk 3 above)
Comment (alallema) [reply scripts/update_meilisearch_version.sh:112]: >Should we exit the script if it fails?

But what happens if you don't set a `MEILISEARCH_MASTER_KEY`? You can if you run your instance in `dev` mode

Comment (bidoubiwa) [reply scripts/update_meilisearch_version.sh]: You are right! My bad :)

**Pull request merged**